src/matlab2tikz.m

```diff
@@ -2026,6 +2044,30 @@ function assertRegularAxes(handle)
                                        markerInfo.options);
         end
     end
+
+    type = getOrDefault(h, 'Type', 'none');
+    if strcmp(type, 'errorbar')
+        %'capSize' -> errorbar marker size
+        capSize = get(h, 'CapSize');
```

Comment:
We need the following here for backward compatibility: 
`capSize = getOrDefault(h, 'CapSize',get(h,'MarkerSize'))`

since `'CapSize'` was introduced in R2016b